src/index.js

```diff
@@ -101,11 +107,17 @@ export default Selector(complexSelector => {
 }).addCustomMethods({
     getVue: (node, fn) => {
         function getData (instance, prop) {
-            const result = {};
-
-            Object.keys(prop).forEach(key => {
-                result[key] = instance[key];
-            });
+            let result = {};
+            
+            if (prop === 'reference') {
```

Comment:
Remove this code from `getData` function.

The `getComponentReference` function code should look like this:
```js
function getComponentReference(instance) {
      return instance.$vnode && instance.$vnode.data && instance.$vnode.data.ref;   
}
```